drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -7,7 +7,11 @@
  */
 
 #define LOG_MODULE_NAME ieee802154_rf2xx
+#if defined(CONFIG_IEEE802154_DRIVER_LOG_LEVEL)
```

Comment:
same as previous patch, no need of ifdef.

Reply:
Rollback